packages/app/src/runner/SpecRunnerHeaderRunMode.vue

```diff
@@ -70,6 +72,14 @@ const displayScale = computed(() => {
 
 const autStore = useAutStore()
 
+const AUTHeaderEl = ref<HTMLDivElement>()
```

Comment:
I was going to suggest sharing this via composable

```ts
function useAutHeader () {
  const autHeaderEl = ref<HTMLDivElement>()
  const { height } = useElementSize(AUTHeaderEl)
  
  watch(height, () => {
    autStore.setSpecRunnerHeaderHeight(height.value)
  })

  return {
    autHeaderEl
  }
}

// usage

const { autHeaderEl } = useAutHeader()
```

But now I think we don't even need to duplicate this. In run mode, the AUT header should never resize, right, since the user cannot drag/drop during run mode - so I don't think you need this code at all here, only in `SpecRunnerHeaderOpenMode.vue`.

Reply:
The header can resize in runmode sometimes, based on the size of the elements in it - a long URL will cause wrapping inside the header, and (less likely of course) a long custom browser name would do the same. 

We have a [ticket](https://cypress-io.atlassian.net/browse/UNIFY-1510) to correct this, but I don't think it's a truly solved problem yet, though maybe, for runmode, the behavior should be different. The wrapping seems way more important/useful when you are resizing the window in the GUI.

Good point about the composable, I think we should do that for now, and I'll add some comments in UNIFY-1510 about pros/cons around how we currently handle wrapping inside the header.